installer/core/providers/aws/validate.py

```diff
@@ -178,6 +203,12 @@ def validate(self):
 
         status = self.validate_user_policies()
 
+        if(Settings.get('MINIMUM_RAM',None) and Settings.get('MINIMUM_STORAGE',None)):
```

Comment:
Both checks should be done seperately. It might be possible to check only RAM or Disk or Both.